modules/fscloud/main.tf

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+module "cbr_zone" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  version          = "1.18.0"
+  name             = var.cbr_zone_name
+  zone_description = "CBR Network zone representing VPC"
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [{
+    type  = "vpc", # to bind a specific vpc to the zone
+    value = var.vpc_crn,
+  }]
+}
+
+module "secrets_manager" {
+  source                           = "../.."
+  resource_group_id                = var.resource_group_id
+  region                           = var.region
+  secrets_manager_name             = var.secrets_manager_name #tfsec:ignore:general-secrets-no-plaintext-exposure
+  sm_service_plan                  = "standard"
```

Comment:
this needs to be a variable, and ensure in our tests we set it to trial (the default value of the module can be standard though)

Reply:
variable added